pokemongo_bot/path_navigator.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+import logger
+from cell_workers.utils import distance, i2f, format_dist
+from human_behaviour import sleep
+from step_walker import StepWalker
+from pgoapi.utilities import f2i
+
+
+class PathNavigator(object):
```

Comment:
In a lot of ways, every navigator does this right? The navigator comes up with a set of points and then walks them. Perhaps we need to find a way for this to be some kind of base that the other navigators can be built on so we don't have repeated code.